packages/babel-preset-default/index.js

```diff
@@ -31,6 +31,7 @@ module.exports = function( api ) {
 		if ( isWPBuild( wpBuildOpts.name ) ) {
 			opts.modules = wpBuildOpts.modules;
 		}
+		opts.shippedProposals = true;
```

Comment:
You can add the flag to the `opts` initial value:
```js
const opts = {
  shippedProposals: true;
}
```
and then set the values that are set only conditionally.

That's a cosmetic change though -- otherwise, the patch is correct.